src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    protected double firstLeg;
+    protected double secondLeg;
```

Comment:
```suggestion
    private double firstLeg;
    private double secondLeg;
```
Please recheck all classes to have proper access modifiers